lib/src/widgets/html_editor_widget_web.dart

```diff
@@ -557,7 +557,7 @@ class _HtmlEditorWidgetWebState extends State<HtmlEditorWidget> {
           window.parent.postMessage(JSON.stringify({"view": "$createdViewId", "message": "$_summernoteLoadedMessage"}), "*");
         }
         window.addEventListener('load', iframeLoaded, false);
-        window.addEventListener('beforeunload', (event) => {
+        window.addEventListener('pagehide', (event) => {
```

Comment:
can you explain about it?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Window/pagehide_event
According to the document, `pagehide` is better than `beforeunload` when detect unload event